src/topology/separation.lean

```diff
@@ -1049,6 +1049,15 @@ lemma is_closed_eq [t2_space α] {f g : β → α}
   (hf : continuous f) (hg : continuous g) : is_closed {x:β | f x = g x} :=
 continuous_iff_is_closed.mp (hf.prod_mk hg) _ is_closed_diagonal
 
+lemma is_open_ne_fun [t2_space α] {f g : β → α}
+  (hf : continuous f) (hg : continuous g) : is_open {x:β | f x ≠ g x} :=
```

Comment:
Would there be any precedent for using `locus` in the name here?

Reply:
Not as far as I can tell, but it would make sense. I agree that the naming is not very good, but `is_open_ne` is already taken. Probably I should also rename `is_closed_eq` to `is_closed_eq_fun` or `is_closed_eq_locus` for coherence, by the way. (I think I still prefer the version with `fun` than with `locus` as it looks more mathlib-standard to me).

Reply:
Except that there are 60 uses of `is_closed_eq` in the library, so I'll only do it if you confirm you think it's a good idea to rename :-)

Reply:
I don't feel strongly either way. Probably if we go down this route we should define `function.eq_locus f g` as `{x:β | f x = g x}`, to match things like `monoid_hom.eq_locus`; then this would be `is_open_compl_locus` or similar; but that' not in scope here!